docs/source/python_api/mlflow.metrics.rst

```diff
@@ -1,7 +1,58 @@
 mlflow.metrics
 ==============
 
+The ``mlflow.metrics`` module helps you quantitatively and qualitatively measure your models. These :py:class:`EvaluationMetric <mlflow.metrics.EvaluationMetric>` are used by the :py:func:`mlflow.evaluate()` API, either computed automatically depending on the ``model_type`` or specified via the ``custom_metrics`` parameter. Evaluation results are stored as :py:class:`MetricValue <mlflow.metrics.MetricValue>` and are logged to the MLflow run.
+
+.. autoclass:: mlflow.metrics.MetricValue
+
+.. autoclass:: mlflow.metrics.EvaluationMetric
+
+We provide the following default :py:class:`EvaluationMetric <mlflow.metrics.EvaluationMetric>` for evaluating text models. 
+
+.. autodata:: mlflow.metrics.ari_grade_level
+   :annotation:
+
+.. autodata:: mlflow.metrics.flesch_kincaid_grade_level
+   :annotation:
+
+.. autodata:: mlflow.metrics.perplexity
+   :annotation:
+
+.. autodata:: mlflow.metrics.rouge1
+   :annotation:
+
+.. autodata:: mlflow.metrics.rouge2
+   :annotation:
+
+.. autodata:: mlflow.metrics.rougeL
+   :annotation:
+
+.. autodata:: mlflow.metrics.rougeLsum
+   :annotation:
+
+.. autodata:: mlflow.metrics.toxicity
+   :annotation:
```

Comment:
Can we name / identify the model type(s) for which each of these built-in metrics are automatically computed?

Reply:
Will address in a followup PR once the regression and classification metrics are also merged.